config/config.yaml

```
@@ -1,13 +1,13 @@
 metric_calculation:
-  run_id: 22.11.1
+  run_id: 22.11.2
 
   data_repositories:
-    release_output_root: gs://open-targets-pre-data-releases/22.11/output/etl/parquet
+    release_output_root: gs://open-targets-pre-data-releases/22.11.2/output/etl/parquet
```

Would it make sense to parametrize the release input folder?
```suggestion
    release_output_root: gs://open-targets-pre-data-releases/${metric_calculation.run_id}/output/etl/parquet
```

Same for row10